pom.xml

```diff
@@ -31,38 +33,82 @@
                 <enabled>false</enabled>
             </releases>
         </repository>
+
     </repositories>
+
     <dependencies>
+
         <dependency>
             <groupId>fr.paris.lutece</groupId>
             <artifactId>lutece-core</artifactId>
-            <version>[7.0.10,7.9.9)</version>
+            <version>8.0.0-SNAPSHOT</version>
             <type>lutece-core</type>
         </dependency>
+ 
+     	<dependency>
+    	 	<groupId>fr.paris.lutece.plugins</groupId>
+			<artifactId>plugin-parser</artifactId>
+    		<version>2.0.0-SNAPSHOT</version>
+            <type>lutece-plugin</type>
+    	</dependency>
+    	
         <!-- Add to resize image -->
         <dependency>
             <groupId>org.imgscalr</groupId>
             <artifactId>imgscalr-lib</artifactId>
             <version>4.1</version>
             <type>jar</type>
         </dependency>
-		<dependency>
-			<groupId>org.codehaus.jackson</groupId>
-			<artifactId>jackson-mapper-asl</artifactId>
-			<version>1.5.0</version>
-		</dependency>
+        
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+		
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-digester3</artifactId>
+            <version>3.2</version>
+        </dependency>
+        
+	<!-- JUnit -->
+	<dependency>
+		<groupId>fr.paris.lutece.plugins</groupId>
+		<artifactId>library-lutece-unit-testing</artifactId>
+		<type>jar</type>
+		<scope>test</scope>
+	</dependency>
+
+	<dependency>
+		<groupId>fr.paris.lutece.plugins</groupId>
+		<artifactId>library-core-utils</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</dependency>
     </dependencies>
     <properties>
         <componentName>document</componentName>
         <jiraProjectName>DOCUMENT</jiraProjectName>
         <jiraComponentId>10166</jiraComponentId>
     </properties>
 
+	<build>
```

Comment:
Do not configure Maven plugins at the plugin level; if there is an issue, it should be reported so it can be fixed in the global POM.